### ISSUE TITLE
the loss of Generator didn't decrease

### ISSUE THREAD

**Issue by ytwas** [znxlwm/pytorch-MNIST-CelebA-cGAN-cDCGAN#9]
I only changed one place
 ` transforms.Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5)) `
to` transforms.Normalize((0.1307,), (0.3081,))`
because it causes error.
following is trainning process:
[1/50] - ptime: 29.42, loss_d: 0.855, loss_g: 1.262
[2/50] - ptime: 29.97, loss_d: 0.288, loss_g: 2.511
[3/50] - ptime: 31.54, loss_d: 0.090, loss_g: 3.608
[4/50] - ptime: 32.44, loss_d: 0.086, loss_g: 4.054
[5/50] - ptime: 34.58, loss_d: 0.070, loss_g: 4.227
[6/50] - ptime: 32.97, loss_d: 0.033, loss_g: 4.767
[7/50] - ptime: 33.55, loss_d: 0.014, loss_g: 5.374
[8/50] - ptime: 34.24, loss_d: 0.029, loss_g: 5.416
[9/50] - ptime: 34.68, loss_d: 0.008, loss_g: 5.843
[10/50] - ptime: 35.56, loss_d: 0.016, loss_g: 5.878
[11/50] - ptime: 38.06, loss_d: 0.006, loss_g: 6.242
[12/50] - ptime: 34.45, loss_d: 0.026, loss_g: 6.199
[13/50] - ptime: 32.97, loss_d: 0.005, loss_g: 6.561
[14/50] - ptime: 32.59, loss_d: 0.015, loss_g: 6.542
[15/50] - ptime: 32.66, loss_d: 0.002, loss_g: 6.981
[16/50] - ptime: 32.96, loss_d: 0.025, loss_g: 6.400
[17/50] - ptime: 33.44, loss_d: 0.019, loss_g: 6.936
[18/50] - ptime: 33.80, loss_d: 0.008, loss_g: 6.489
[19/50] - ptime: 34.30, loss_d: 0.002, loss_g: 7.097
[20/50] - ptime: 34.98, loss_d: 0.040, loss_g: 6.319
[21/50] - ptime: 34.86, loss_d: 0.003, loss_g: 6.906
[22/50] - ptime: 34.65, loss_d: 0.003, loss_g: 7.332
[23/50] - ptime: 34.74, loss_d: 0.008, loss_g: 7.447
[24/50] - ptime: 34.87, loss_d: 0.003, loss_g: 7.552
[25/50] - ptime: 34.79, loss_d: 0.001, loss_g: 7.913
[26/50] - ptime: 35.62, loss_d: 0.009, loss_g: 7.999
[27/50] - ptime: 37.91, loss_d: 0.006, loss_g: 7.409
[28/50] - ptime: 33.79, loss_d: 0.001, loss_g: 8.188
[29/50] - ptime: 32.57, loss_d: 0.001, loss_g: 8.127
learning rate change!
[30/50] - ptime: 32.99, loss_d: 0.005, loss_g: 7.739
[31/50] - ptime: 33.69, loss_d: 0.001, loss_g: 7.975
[32/50] - ptime: 34.34, loss_d: 0.001, loss_g: 8.114
[33/50] - ptime: 35.43, loss_d: 0.001, loss_g: 8.140
[34/50] - ptime: 36.85, loss_d: 0.002, loss_g: 7.763
[35/50] - ptime: 40.14, loss_d: 0.001, loss_g: 8.218
[36/50] - ptime: 44.25, loss_d: 0.001, loss_g: 8.467
[37/50] - ptime: 45.44, loss_d: 0.001, loss_g: 8.708
[38/50] - ptime: 40.60, loss_d: 0.001, loss_g: 8.666
[39/50] - ptime: 40.33, loss_d: 0.001, loss_g: 8.824
learning rate change!
[40/50] - ptime: 39.38, loss_d: 0.000, loss_g: 8.775
[41/50] - ptime: 37.45, loss_d: 0.000, loss_g: 8.842
[42/50] - ptime: 37.45, loss_d: 0.001, loss_g: 8.971
[43/50] - ptime: 38.80, loss_d: 0.001, loss_g: 8.718
[44/50] - ptime: 38.23, loss_d: 0.000, loss_g: 8.672
[45/50] - ptime: 40.54, loss_d: 0.000, loss_g: 8.708
[46/50] - ptime: 36.43, loss_d: 0.000, loss_g: 8.707
[47/50] - ptime: 34.01, loss_d: 0.000, loss_g: 8.800
[48/50] - ptime: 36.30, loss_d: 0.000, loss_g: 8.910
[49/50] - ptime: 36.61, loss_d: 0.000, loss_g: 8.990
[50/50] - ptime: 35.31, loss_d: 0.000, loss_g: 9.075
Avg one epoch ptime: 35.55, total 50 epochs ptime: 1881.31
Training finish!... save training results
